src/web/src/components/Sites/SitesForms/Photos.vue

```diff
@@ -0,0 +1,173 @@
+                                    label="Caption"
+                                    required
+                                    ></v-text-field>
+
+                                    <v-text-field 
+                                    v-model="item.comments"
+                                    label="Comments"
+                                    required
+                                    ></v-text-field>
+
+                                    <v-text-field 
+                                    v-model="item.creditLine"
+                                    label="CreditLine"
+                                    required
+                                    ></v-text-field>
+
+                                    <v-text-field 
+                                    v-model="item.location"
+                                    label="Location"
+                                    required
+                                    ></v-text-field>
+                                    <v-file-input
+                                        :id="`fi-${i}`"
+                                        label="Upload image"
+                                        prepend-icon="mdi-camera"
+                                        accept="image/*"
+                                        @change="onFileSelection($event, i)"
+                                    ></v-file-input>
+                                </v-col>
+                            </v-row>
+                        </v-alert>
+                    </v-col>
+                    <v-col cols="12">
+                        <v-btn
+                        outlined
+                        color="primary"
+                        @click="addItem('photos')"
+                        >
+                            Add New
+                        </v-btn>
+                    </v-col>
+
+                </v-row>
+              </v-container>
+
+            </v-form>
+        </div>
+</template>
+
+<script>
+/* Important**, field data that was not found on the swaggerhub api docs provided was assumed to be in development, hence, some placeholder variables were created. */
+export default {
+    name: "formPhotos",
+    data: () => ({
+            valid: false,
+            generalRules: [
+                v => !!v || 'This input is required',
+                v => v.length <= 20 || 'This input must be less than 20 characters',
+            ],
+            /* Placeholder variables below this line **Read above** */
+            categoryOfProperty: '',
+            fields:{
+                photos: [{id: '', name:'', caption: '', comments:'', creditLine: '', location:'', img: null},
+                {id: '', name:'', caption: '', comments:'', creditLine: '', location:'', img:null}],
+            }
+
+
+    }),
+    methods:{
+        removeItem(objName, position){
+            if (position > -1) {
+              this.fields[objName].splice(position, 1);
+            }
+        },
```

Comment:
The `removeItem()` pattern is repeated a lot. Can it be refactored out for more DRY?